src/sentry/notifications/notifications/activity/assigned.py

```diff
@@ -8,7 +8,7 @@
 
 
 class AssignedActivityNotification(GroupActivityNotification):
-    referrer_base = "assigned-activity"
+    metrics_key = "assigned_activity"
```

Comment:
@mgaeta I like the idea of combining everything into one field, but maybe we could pick a different variable name? Like `notification_slug` or something, not sure. But since this field is used for more than metrics, I think the variable name should match.

Reply:
Maybe `notification_category`?

Reply:
Maybe `category`?

Reply:
I prefer this key to only be used for internal metrics. The description says:
> When we want to collect analytics about this type of notification, we will use this key.

Reply:
@mgaeta I'm confused, a change in this PR is making `metrics_key` be used for the referrer query param: https://github.com/getsentry/sentry/pull/33426/files#diff-cc93601c6e7207ef811a9e5b844d97a04dcce9f5e740cb0666680976888e587fR145

I'm fine with a single source of truth for these fields (I think it's a good optimization), just need variable names to make sense

Reply:
The `referrer` query parameter is being used to collect metrics.

Reply:
Sure, fair enough. We can leave it as `metrics_key`